docs/admin/setup/distributed.rst

```diff
@@ -617,8 +617,12 @@ These credentials are used to clone repositories via HTTPS. You must also add th
                 container-cleanup:
                     expiry-minutes: 5                               # Time after a hanging container will automatically be removed
                     cleanup-schedule-minutes: 60                    # Schedule for container cleanup
+                build-agent:
+                    short-name: "artemis-build-agent-X"             # Short name of the build agent. This should be unique for each build agent. Only lowercase letters, numbers and hyphens are allowed.
```

Comment:
Do I get it right that the default config fails due to the "X" to ensure the value has to be changed?

Reply:
Yes. Do you think it's better if I put a valid value there?

Reply:
So I am fine if we leave it as it is, just wanted to make sure it is intended :)